### PR TITLE
add debian10 arm64 slaves

### PR DESCRIPTION
**Pull request by jbergler**
I've noticed that there has been a small handful of issues with GPL
violations on arm64 over the past year or so. It seems like this is
something we can catch by leveraging the AWS graviton arm64 instances
for build slaves.

### CODE REVIEW

**Comment by jbergler** [comment]
I'll call out right at the get go that I have no idea how to test this change, since I think a lot of the magic for actually running the buildbot setup requires access to some ec2 accounts and more understanding than I was able to piece together from the docs.

I'm happy to spend more time iterating on this to get it right if someone can provide me the guidance on what I can do to be more confident about the change.

**Comment by jbergler** [comment]
Some examples of the types of issues I'm hoping this will help catch:
- https://github.com/openzfs/zfs/issues/10985
- https://github.com/openzfs/zfs/issues/8545

File: master/buildslaves.py [hunk 1]
```diff
@@ -217,9 +217,14 @@ def __init__(self, name, **kwargs):
 
 # Create an HVM EC2 large latent build slave
 class ZFSEC2BuildSlave(ZFSEC2Slave):
-    def __init__(self, name, **kwargs):
+    def __init__(self, name, arch="amd64", **kwargs):
+        instance_types = {
+            "amd64": "c5d.large",
+            "arm64": "a1.xlarge"
```
Comment (jbergler) [inline master/buildslaves.py:223]: These instances are only available in a few regions for now (us-west-2 seems the closest to everything else)
How should this be handled?

Comment (jbergler) [reply master/buildslaves.py]: fixed in f197c4e

Comment (jbergler) [reply master/buildslaves.py]: I lied, c6g is available in us-west-1 but it lacks ephemeral storage.
c6gd has nvme storage available but is [only available](https://aws.amazon.com/about-aws/whats-new/2020/07/announcing-new-amazon-ec2-instances-powered-aws-graviton2-processors/) in us-east-* and us-west-2

Comment (behlendorf) [reply master/buildslaves.py]: The way things are now we'll need it to be available in us-west-1.  But if we're only doing compilation testing we don't need anything fancy.

**Comment by behlendorf** [comment]
I completely agree we should start using the graviton arm64 instances to catch this kind of thing.  In fact there's a similar change for FreeBSD #199 pending.  Unfortunately testing this kind of change is a bit problematic.  I'm swamped right now, but I can probably test it out in a few weeks.

File: master/buildslaves.py (hunk 1 above)
Comment (jbergler) [reply master/buildslaves.py]: Ok, I switched to c6g which is available in us-west-1 but lacks NVMe and updated the bootstrap script to handle the case where those were missing.

**Comment by jbergler** [comment]
> I completely agree we should start using the graviton arm64 instances to catch this kind of thing. In fact there's a similar change for FreeBSD #199 pending. Unfortunately testing this kind of change is a bit problematic. I'm swamped right now, but I can probably test it out in a few weeks.

No worries, I pushed some more changes from testing the bootstrap scripts manually.
I get to the point where buildbot starts but fails to connect because it doesn't have valid credentials, so hopefully that bodes well for when you have some time.

**Comment by jbergler** [comment]
Hey @behlendorf, no rush, just checking in here to see if you've had a chance to look at this at all / if there was anything else I can do?

**Comment by behlendorf** [comment]
@jbergler the change looks reasonable but I haven't yet had a chance to give it a test.  Hhopefully in the next couple of days, sorry about the delay.

**Comment by behlendorf** [comment]
Merged!  This change worked out nicely with just a few minor tweaks.  Thanks for putting this together, I've gone ahead and rolled out the new [Debian 10 arm64 builder](http://build.zfsonlinux.org/builders/Debian%2010%20arm64%20%28BUILD%29).

**Pull request closed without merging**